src/helpers.js

```diff
@@ -0,0 +1,55 @@
+const sha1 = require('sha1');
+
+export const checksum = str => parseInt(sha1(str).substr(0, 8), 16);
+
+export const getLastItem = obj => {
+  const keys = Object.keys(obj);
+  const lastKey = keys[keys.length - 1];
+
+  const item = obj[lastKey];
+  delete obj[lastKey];
+
+  return item;
+};
+
+export const getSql = () => {
+  let sql;
+  if (process.env.APP_ENV === 'browser') {
+    require('script!sql.js');
+    sql = window.SQL;
+  } else {
+    sql = require('sql.js');
+  }
+  return sql;
+};
+
+export const rand = () => Math.random() * 100000000 | 0;
+
+export const getDb = () => new (getSql().Database)();
+
+export const getZip = (...args) => {
```

Comment:
Also, I would like to clarify the origin of all this. Is it because I said "Contributing lacking tests first is definitely a good choice" https://github.com/ewnd9/anki-apkg-export/pull/5#issuecomment-263160745 ? :no_mouth: I'm sorry, I meant it only in context of 4 commented lines at https://github.com/ewnd9/anki-apkg-export/commit/3c16fced9eb01185fc46bdf577626440af9dc959#diff-c1129c8b045390789fa8ff62f2c6b4a9L29